django_irods/storage.py

```diff
@@ -99,15 +99,17 @@ def zipup(self, in_name, out_name):
         # SessionException will be raised from run() in icommands.py
         self.session.run("ibun", None, '-cDzip', '-f', out_name, in_name)
 
-    def unzip(self, zip_file_path):
+    def unzip(self, zip_file_path, unzipped_folder=None):
         """
         run iRODS ibun command to unzip files into a new folder
         :param zip_file_path: path of the zipped file to be unzipped
         :return: the folder files were unzipped to
         """
 
         abs_path = os.path.dirname(zip_file_path)
-        unzipped_folder = os.path.splitext(os.path.basename(zip_file_path))[0].strip()
+        if(unzipped_folder is None):
```

Comment:
@sblack-usu Change the condition to ```if not unzipped_folder:``` so it can cover the case when an empty string is being passed in as well.

Reply:
good idea, done